src/concepts/concepts_readme.md

```diff
@@ -0,0 +1,16 @@
+# ThreeFold Grid Concepts
+
+On this section we will explore the fundamental principles and concepts behind the ThreeFold Grid. This comprehensive resource will take you on a journey through the core technologies that underpin the ThreeFold Grid, empowering you to understand and leverage the decentralized nature of this groundbreaking infrastructure.
+
+## Learn the Basics
+
+- [TFGrid Primitives](./grid_primitives.md)
+- [Proof of Utilization](./proof_of_utilization.md)
+- [Contract Grace Period](./contract_grace_period.md)
+- [Buying and Storing TFTs](./buying_storing_tft.md)
+
+## Take an In-depth Looks
+
+- [TF Library - Technology](https://library.threefold.me/info/threefold#/technology/threefold__technology)
+- [What's New on TFGrid v3.x](./grid3_whatsnew.md)
+- [TF Library - Glossary](https://library.threefold.me/info/manual/#/manual__defs)
```

Comment:
technology is linked to the **threefold wiki** **(library.threefold.me) / info_threefold_pub.** 
its different to the old manual, old manual is manual.threefold.io / info_manual3

yes, glossary still linked to old manual, but since this migration is a long process, i think for now its ok until we move everything in.